.rubocop_todo.yml

```diff
@@ -1,137 +1,476 @@
-# This configuration was generated by `rubocop --auto-gen-config`
-# on 2014-07-21 12:48:09 +0200 using RuboCop version 0.24.1.
+# This configuration was generated by
+# `rubocop --auto-gen-config`
+# on 2016-02-08 14:42:45 +0100 using RuboCop version 0.37.0.
 # The point is for the user to remove these configuration records
 # one by one as the offenses are removed from the code base.
 # Note that changes in the inspected code, or installation of new
 # versions of RuboCop, may require this file to be generated again.
 
-# Offense count: 3
+# Offense count: 2
+Lint/DuplicatedKey:
+  Exclude:
+    - 'db/migrate/20140801164901_move_conference_media_to_commercial.rb'
+    - 'db/migrate/20140801170430_move_event_media_to_commercial.rb'
+
+# Offense count: 4
+Lint/IneffectiveAccessModifier:
+  Exclude:
+    - 'app/models/commercial.rb'
+    - 'app/models/conference.rb'
+
+# Offense count: 84
+Metrics/AbcSize:
+  Max: 75
+
+# Offense count: 7
+# Configuration parameters: CountComments.
+Metrics/ClassLength:
+  Max: 514
+
+# Offense count: 12
+Metrics/CyclomaticComplexity:
+  Max: 12
+
+# Offense count: 956
+# Configuration parameters: AllowHeredoc, AllowURI, URISchemes.
+# URISchemes: http, https
+Metrics/LineLength:
+  Max: 208
+
+# Offense count: 100
+# Configuration parameters: CountComments.
+Metrics/MethodLength:
+  Max: 56
+
+# Offense count: 1
+# Configuration parameters: CountComments.
+Metrics/ModuleLength:
+  Max: 256
+
+# Offense count: 7
+Metrics/PerceivedComplexity:
+  Max: 15
+
+# Offense count: 1
 # Cop supports --auto-correct.
+Performance/StringReplacement:
+  Exclude:
+    - 'spec/support/save_feature_failures.rb'
+
+# Offense count: 13
+# Cop supports --auto-correct.
+# Configuration parameters: EnforcedStyle, SupportedStyles, Include.
+# SupportedStyles: action, filter
+# Include: app/controllers/**/*.rb
+Rails/ActionFilter:
+  Exclude:
```

Comment:
Because they have offenses? The general idea of the rubocop_todo.yml is that you record which offenses you have where and then you can either explicitly disable or enable the cops in your rubocop.yml and remove the setting in rubocop_todo.yml